lib/pacemaker/pcmk_sched_messages.c

```diff
@@ -47,7 +47,7 @@ log_resource_details(pe_working_set_t *data_set)
         }
     }
 
-    g_list_free_full(all, free);
+    g_list_free(all);
```

Comment:
I don't see an advantage in the wrapper, it doesn't save any lines, and singleton is typically used to mean that every call returns the same list

Getting rid of the strdup()'s where possible is still a good idea

Reply:
Fixed.